javascript/javascript2/week3/script3.js

```diff
@@ -0,0 +1,64 @@
+const btnStart = document.querySelector("button");
```

Comment:
You can change the style of this btn at the game over and make the possible to re-start the game. 
Of course, you'll need to clean up the data in the count and so forth.
Just an idea to play around with.